### PR TITLE
dev/backend

### PR DESCRIPTION
**Pull request by LIKEROFJAZZ**
basic functionality of Flask API for post requests

### CODE REVIEW

File: api/flask_api.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+app = Flask(__name__)
+api = flask_restful.Api(app)
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+class email(Resource):
+    def post(self):
+        return {'message': "posted successfully"}, 200
+
```
Comment (valdelaseras) [inline api/flask_api.py:19]: I require changes. Remove this empty line here

File: web/scripts/index.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ const emailInput = document.getElementById('email-input');
 const submitBtn = document.getElementById('submit-btn');
 const form = document.getElementById('form');
 
-const host = window.location.protocol + window.location.host;
+const host = 'http://127.0.0.1:8001';
```
Comment (valdelaseras) [inline web/scripts/index.js:5]: We need to make sure this isnt hardcoded later on

File: web/scripts/index.js [hunk 3]
```diff
@@ -76,7 +77,7 @@ post = ( url, data ) => {
             console.log(xhr.responseText);
         }};
 
-    xhr.send( data );
+    xhr.send( JSON.stringify(data) );
```
Comment (valdelaseras) [inline web/scripts/index.js:80]: we should also check what the JSON looks like when the backend receives it, we only checked if the POST was working

**Review by valdelaseras — COMMENTED** [review]
left some comments for demo purpose

File: api/flask_api.py (hunk 1 above)
Comment (LIKEROFJAZZ) [reply api/flask_api.py:19]: I refuse. down with the patch-riarchy

**Pull request merged**